### DISCUSSION TITLE
Example of CSS for custom style on Paragraph?

### DISCUSSION THREAD

**Discussion by edmondop** [brechtm/rinohtype#346] [Q&A] [answered]
I am trying to modify the font size for a paragraph with a specific style, however I don't find the right syntax.

Take this example https://github.com/brechtm/rinohtype/blob/7b03425b327d7308a899d503363e466cb2346db6/src/rinoh/template.py#L343, I didn't find any matching CSS rule in the docs. Can anyone help?

**Comment by brechtm** [comment, marked as answer]
The [default matcher](http://www.mos6581.org/rinohtype/master/elementstyling.html#matchers) defines a style label for _Paragraph_ with `style='title page subtitle'`. The style label is also named _title page subtitle_. So for this paragraph, you can create a style definition in your style sheet like this:

```ini
[title page subtitle]
base = body
font_size = 10pt
```

Have a look at the [Sphinx style sheet](http://www.mos6581.org/rinohtype/master/sphinx_stylesheet.html) to get an idea of which style labels are available and the style properties they support. See the [reference documentation](http://www.mos6581.org/rinohtype/master/stylesheets.html) for the full list of style labels and the properties supported by each of the style classes.

If you introduce _Paragraph_ instances with a custom _style_ in your template; e.g. `Paragraph('hello', style='my style')`, you need to include a [selector](http://www.mos6581.org/rinohtype/master/elementstyling.html#custom-selectors) in the style sheet:

```ini
[my paragraph : Paragraph('my style')]
base = body
font_size = 10pt
```

Within the brackets, '_my paragraph_' is the style label, and '_my style_' should match what is passed to _Paragraph_'s _style_ argument.
